darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -239,9 +239,27 @@ def validation_step(self, val_batch, batch_idx) -> torch.Tensor:
             prog_bar=True,
             sync_dist=True,
         )
-        self._calculate_metrics(output, target, self.val_metrics)
+        self._update_metrics(output, target, self.val_metrics)
         return loss
 
+    def _compute_metrics(self, metrics):
+        res = metrics.compute()
```

Comment:
Can you move this method to where method `_update_metric()` is defined?

```suggestion
        if not len(metrics):
            return

        res = metrics.compute()
```

Reply:
Moved and "skipped" when needed. For some reason linter doesn't add empty line by default